examples/HelloScalaApp/hellolibrary/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.library'
+apply plugin: 'android-scala-support'
+
+android {
+	compileSdkVersion 24
+	buildToolsVersion "24.0.2"
+
+	lintOptions {
+		abortOnError false
+	}
+
+	defaultConfig {
+		minSdkVersion 14
+		targetSdkVersion 24
+		versionCode 1
+		versionName "1.0"
+
+		testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+	}
+
+	buildTypes {
+		release {
+			minifyEnabled false
+			proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+		}
+	}
+}
+
+androidScala {
+	scalaVersion '2.11.8'
+	zincVersion '0.3.12'
+	addScalaLibrary true
```

Comment:
have we not agreed that this doesn't make sense? what is the scenario when you dont want to add scala library? even if this scenario existed, library should be added by default and excluding it should be explicit